packages/arb-token-bridge-ui/src/components/TransferPanel/NetworkListbox.tsx

```diff
@@ -83,35 +77,25 @@ export function NetworkListbox({
         <Listbox.Options className="absolute z-20 ml-2 mt-2 overflow-hidden rounded-xl bg-white shadow-[0px_4px_12px_#9e9e9e]">
           {options.map((option, index) => {
             return (
-              <Tooltip
+              <Listbox.Option
```

Comment:
Removed tooltip and disabled props. This was only used to disable one and nova selections but we show a dialog now instead.